src/core/symbology/qgsmarkersymbollayer.h

```diff
@@ -613,8 +613,9 @@ class CORE_EXPORT QgsSvgMarkerSymbolLayer : public QgsMarkerSymbolLayer
      * \param context symbol render context
      * \param scaledSize size of symbol to render
      * \param hasDataDefinedAspectRatio will be set to TRUE if marker has data defined aspectRatio
+     * \param scaledHeight will be set to the resulting height taking size (i.e. width) and ratio into account
      */
-    double calculateAspectRatio( QgsSymbolRenderContext &context, double scaledSize, bool &hasDataDefinedAspectRatio ) const;
+    double calculateAspectRatio( QgsSymbolRenderContext &context, double scaledSize, bool &hasDataDefinedAspectRatio, double &scaledHeight ) const;
```

Comment:
```suggestion
    double calculateAspectRatio( QgsSymbolRenderContext &context, double scaledSize, bool &hasDataDefinedAspectRatio, const double &scaledHeight ) const;
```

Shouldn't it be const if passed by value?
And could it have a default value = 0?